scripts/schemas/arch-schema.json

```
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://zephyrproject.org/schemas/arch-schema.json",
```

Prefer to add the project in the id, as I'd like to do the same for `west`

```suggestion
  "$id": "https://zephyrproject.org/schemas/zephyr/arch-schema.json",
```